pkg/component/operator/audio/v0/audio.go

```diff
@@ -0,0 +1,13 @@
+package audio
+
+type Audio string
```

Comment:
I suspect this is an exported type because it conflicts with the `audio` package, correct me if I'm wrong.
- Can we use another alias so the type is unexported?
- Do we need the alias at all? The type doesn't implement any extra methods over `string`.

Reply:
Will rename it to `base64Audio` to align with the Image operator too. These are temporary data structure anyhow before we have the binary data exchange implementation.